src/pages/HomePage/HomePageComponents/NavBar/NavBar.css

```diff
@@ -1,87 +1,94 @@
 .navigation-container {
-    padding: 1rem 0;
+  padding: 0 0.5rem;
+  background-color: var(--white);
 }
 
-.logo-input {
-    display: flex;
-    width: 70%;
-    margin-left: 1.5rem;
+.site-logo {
+  width: 12rem;
+}
+
+.header-shop-link {
+  text-decoration: none;
+  color: var(--black);
+}
+
+.header-shop-link:hover {
+  color: var(--primary-color);
 }
 
 .search-input-container {
-    display: flex;
-    align-items: center;
-    width: 60%;
-    padding: 0 1rem;
+  display: flex;
+  align-items: center;
+  width: 30rem;
+  padding: 0 1rem;
+  margin-left: 2rem;
 }
 
 .input-icon {
-    position: relative;
-    right: 3rem;
-    padding: 5px;
+  position: relative;
+  right: 3rem;
+  padding: 5px;
 }
 
 .header-input {
-    padding: 13px;
-    width: 90%;
-    border-radius: 3rem;
-    border: none;
+  padding: 9px;
+  width: 90%;
+  border-radius: 3rem;
+  border-radius: 5px;
```

Comment:
Applied border radius twice

Reply:
Corrected it, Thanks for pointing out 👍🏻